latex-report/main.tex

```diff
@@ -188,35 +188,35 @@
         \item \textbf{Google Cloud Platform} \newline
-        Google Cloud Platform (GCP), offered by Google, is a suite of cloud computing services that runs on the same infrastructure that Google uses internally for its end-user products, such as Google Search, Gmail, file storage, and YouTube.
         \item \textbf{Gunicorn} \newline
-        "Green Unicorn" is a Python Web Server Gateway Interface (WSGI) HTTP server. It is a pre-fork worker model, ported from Ruby's Unicorn project. The Gunicorn server is broadly compatible with a number of web frameworks, simply implemented, light on server resources and fairly fast.
-        \item \textbf{Visual Studio Code} \newline
-        It is a free source-code editor made by Microsoft for Windows, Linux and macOS. Features include support for debugging, syntax highlighting, intelligent code completion, snippets, code refactoring, and embedded Git.
-        \item \textbf{PyCharm Community Edition} \newline
-        PyCharm is an integrated development environment (IDE) used in computer programming, specifically for the Python language. It is developed by the Czech company JetBrains.
         \item \textbf{\LaTeX} \newline
-        This project report was made using \LaTeX. \LaTeX is a high-quality typesetting system; it includes features designed for the production of technical and scientific documentation. \LaTeX is the de facto standard for the communication and publication of scientific documents.
         \item \textbf{Adobe Photoshop} \newline
-        Adobe Photoshop is a raster graphics editor developed and published by Adobe Inc. for Windows and macOS.
+        
     \end{enumerate}
     
     
   \newpage
   \section{About the Project}
   The aim of the project, Sudoku Webapp, is to provide an aesthetically rich interface for sudoku enthusiasts to enjoy sudoku puzzles.\newline
 
-  A website with 3 webpages: home page, sudoku page and leaderboard page, is the frontend of the project with which the user interacts. The webpages are made using Django, HTML, JavaScript and a CSS framework called Tailwind. A MySQL database is the primary data store, and is used to store the leaderboard data, and the sudoku game boards that are served on the site.\newline
+  A website with 3 webpages - home page, sudoku page and leaderboard page - is the frontend of the project with which the user interacts. The entire backend is handled using python and the django framework. The sudoku game boards are generated using a combination of custom classes and functions as well as functions from the random module in python. These game boards are passed onto the website using queues. A MySQL database is the primary data store which is used to store the leaderboard data, and the sudoku game boards that are served onto the site. The webpages are made using HTML, JavaScript and a CSS framework called TailwindCSS. \newline
```

Comment:
Capitalize `Python` and `Django` here I'd say; and if possible everywhere else they're used. (Not very high priority though)

Reply:
Done